content/material_and_questions.md

```diff
@@ -44,4 +53,153 @@ In the below table, any minimum or maximum variation between material years are
+- The identification of a singular pronoun does not meet the above requirement.
+- The identification of a plural pronoun does meet the above requirement.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+If the multiple answers contain at least one "positive" answer AND at least one "negative" answer, this is deemed invalid as a Multiple Answer question.
+The Question Type abbreviation is "MA"
+
+#### Reference
+Used to distinguish exact same words or phrases from the material.
+The entire question is part of the required question + answer.
+If a quizzer needs to provide the reference question, the provided reference question must: 
+- Be of the exact same type as the one on the card (CR, CVR, CRMA, CVRMA).  The quizmaster and/or answer judge must check to confirm this.
+- Have content AND wording is similar to the content and wording of the written question.
+    - For greater clarity, we have provided the following example: CVR According to Luke chapter 23: 37 King of what?  A. The Jews The quizzer can give the question “King of what?” or “King of whom?”, as Jews could be referring to either nation (what) or people (whom).  In this case the quizzer could also give “King of the what?”
+- Have the determining word or phrase from the quizmaster’s card. 
+    - In the 50/50 example, “What men?”, and “Lawless what?, the questions are not similar.
+The determination of a valid Reference question, and the valid sub-type only considers the content in the question. 
+
+##### Chapter-Verse
+Must have a single answer.
+The question must contain a word or phrase that exists more than once in the same chapter.
+Context is the single verse.
+The Question Type abbreviation is "CVR"
+
+##### Multiple-Answer, Chapter-Verse
+Must have more than one answer.
+The Question Type abbreviation is "MACVR"
+
+##### Chapter (only)
+The question must contain a word or phrase that exists more than once in the material.
+The Question Type abbreviation is "CR"
+
+##### Multiple-Answer, Chapter (only)
+Must have more than one answer.
+The Question Type abbreviation is "MACR"
+
+#### Quote
+Will be direct quotes from Scripture
+Must be quoted word-perfect.
+Must be strong enough to stand on its own without requiring additional verses to explain it.
+Must be spiritually significant.
+The entire reference is part of the required question + answer.
+Quizzers may go back and correct their answers if done in the 30-second time limit.  The quizzer will be required to make one full correct rotation. However, the quizzer can be called correct if they have made a mistake only in one area of the verse AND correct the mistake making it abundantly clear to the quiz master and answer judge that they know where their mistake was, and what he corrected words are.
+Context is defined as the verses or verses that are required.
+The quizzer may not continue any non-required verse or verses.
```

Comment:
Did you add this rule? I can't find it in the rulebook.

Reply:
It is a generalized rewrite of the language under Correct Answers applying to F/Q: "The quizzer may not continue into or change to another verse."

If there is a requirement that currently exists under Correct Answers, but applies to a subset of the Question Types, I am moving those to the Question Types definitions, so I can generally say under Correct Answers, "must meet all specific Question Type reqs"

Reply:
WFM.